internal/resources/syntheticmonitoring/resource_check.go

```diff
@@ -672,11 +672,11 @@ multiple checks for a single endpoint to check different capabilities.
 			},
 			"frequency": {
 				Description: "How often the check runs in milliseconds (the value is not truly a \"frequency\" but a \"period\"). " +
-					"The minimum acceptable value is 1 second (1000 ms), and the maximum is 120 seconds (120000 ms).",
+					"The minimum acceptable value is 10 second (10000 ms), and the maximum is 1 hour (3600000 ms).",
 				Type:         schema.TypeInt,
 				Optional:     true,
 				Default:      60000,
-				ValidateFunc: validation.IntBetween(1000, 120000),
+				ValidateFunc: validation.IntBetween(10000, 3600000),
```

Comment:
```suggestion
				ValidateFunc: validation.IntBetween(1000, 3600000),
```

keep at 1s for retrocompatibility